src/torznab.ts

```diff
@@ -20,23 +20,35 @@ import {
 	reformatTitleForSearching,
 	stripExtension,
 } from "./utils.js";
+import { grabArrId } from "./arr.js";
 
 interface TorznabParams {
-	t: "caps" | "search" | "tvsearch" | "movie";
+	t?: "caps" | "search" | "tvsearch" | "movie";
```

Comment:
Because of how i build and use `...ids` for the id caps. I didn't want to create an entirely new interface just for the two types when making t optional provided me a usable interface already.

Reply:
It shouldn't be too hard to make a separate type.